modules/imgproc/src/shapedescr.cpp

```diff
@@ -43,64 +43,39 @@
 namespace cv
 {
 
-// inner product
+/* inner product
+unused
```

Comment:
I think it would be better to remove the unused code completely

Reply:
Ok.